firefox_ui_harness/runners/base.py

```diff
@@ -24,7 +24,7 @@
     'browser.search.update': False,
     'browser.sessionstore.resume_from_crash': False,
     'browser.shell.checkDefaultBrowser': False,
-    'browser.startup.page': 0,
+    'browser.startup.page': 3,
```

Comment:
browser.startup.page: 3 restores the previous session. I added it here when I was initially troubleshooting restart... it's gone now.